charts/loki-distributed/values.yaml

```diff
@@ -162,8 +162,12 @@ loki:
         cache:
           {{- if .Values.memcachedFrontend.enabled }}
           memcached_client:
-            addresses: dnssrv+_memcached-client._tcp.{{ include "loki.memcachedFrontendFullname" . }}.{{ .Release.Namespace }}.svc.{{ .Values.global.clusterDomain }}
             consistent_hash: true
+            host: {{ include "loki.memcachedFrontendFullname" . }}
+            max_idle_conns: 16
+            service: memcached-client
+            timeout: 500ms
+            update_interval: 1m
```

Comment:
This is already default. No point of setting it here.

Reply:
Remove default values.